--- Makefile	2003-12-20 10:25:46.000000000 +0100
+++ Makefile.oden	2008-12-19 16:46:01.000000000 +0100
@@ -40,7 +40,7 @@
 MKDIR		= mkdir
 INSTALL		= install
 STATIC_LIBS	= libipvs/libipvs.a
-
+LDFLAGS		= 
 
 #####################################
 # No servicable parts below this line
@@ -83,7 +83,7 @@
 		make -C libipvs
 
 ipvsadm:	$(OBJS) $(STATIC_LIBS)
-		$(CC) $(CFLAGS) -o $@ $^ $(LIBS)
+		$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^ $(LIBS)
 
 install:        all
 		if [ ! -d $(SBIN) ]; then $(MKDIR) -p $(SBIN); fi
