diff -urN ipvsadm-1.24/Makefile ipvsadm-1.24-patched/Makefile
--- ipvsadm-1.24/Makefile	2003-12-20 19:25:46.000000000 +1000
+++ ipvsadm-1.24-patched/Makefile	2012-10-23 16:23:21.637040490 +1100
@@ -87,7 +87,7 @@
 
 install:        all
 		if [ ! -d $(SBIN) ]; then $(MKDIR) -p $(SBIN); fi
-		$(INSTALL) -m 0755 -s ipvsadm $(SBIN)
+		$(INSTALL) -m 0755 ipvsadm $(SBIN)
 		$(INSTALL) -m 0755 ipvsadm-save $(SBIN)
 		$(INSTALL) -m 0755 ipvsadm-restore $(SBIN)
 		[ -d $(MAN) ] || $(MKDIR) -p $(MAN)
